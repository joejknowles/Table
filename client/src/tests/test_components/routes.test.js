import Routes from '../../components/routes';
import React from 'react';
import ReactDOM from 'react-dom';

import ReactTestUtils from 'react-addons-test-utils';
const shallowRenderer = ReactTestUtils.createRenderer();

it('component renders as before', () => {
  const component = shallowRenderer.render(<Routes />);
  expect(component).toMatchSnapshot();
});
