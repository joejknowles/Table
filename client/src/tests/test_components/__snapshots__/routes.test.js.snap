exports[`test component renders as before 1`] = `
<Router
  history={
    Object {
      "createHref": [Function],
      "createKey": [Function],
      "createLocation": [Function],
      "createPath": [Function],
      "getCurrentLocation": [Function],
      "go": [Function],
      "goBack": [Function],
      "goForward": [Function],
      "listen": [Function],
      "listenBefore": [Function],
      "push": [Function],
      "replace": [Function],
      "transitionTo": [Function],
    }
  }
  render={[Function]}>
  <Route
    component={[Function]}
    path="/" />
  <Route
    component={[Function]}
    path="/play" />
  <Route
    component={[Function]}
    path="/table" />
  <Route
    component={[Function]}
    path="/*" />
</Router>
`;
